src/main/java/com/checkmarx/flow/utils/ScanUtils.java

```diff
@@ -236,6 +200,8 @@ else if (filtersObj != null){
      * @return
      */
     public static ScanRequest overrideCxConfig(ScanRequest request, CxConfig override, FlowProperties flowProperties, JiraProperties jiraProperties){
+        Map<String,String> overridePropertiesMap = new HashMap<>();
```

Comment:
It would be nice to split overrideCxConfig into smaller methods to make it more readable.